man-roxygen/opt-slide-params.R

```diff
@@ -1,5 +1,10 @@
-#' @param col_names A single tidyselection or a tidyselection vector of the
-#'  names of one or more columns for which to calculate the rolling mean.
+#' @param col_names A character vector OR a
+#'  <[`tidy-select`][dplyr_tidy_select]> of the names of one or more columns
+#'  for which to calculate a rolling computation. If a tidy-selection, one
+#'  or more unquoted expressions separated by commas. Variable names can be
+#'  used as if they were positions in the data frame, so expressions like
+#'  `x:y` can be used to select a range of variables. The tidy-selection
+#'  cannot be used to provide output column names.
```

Comment:
The interface does seem to support a character vector of col names passed as-is. Is that somehow unintentional?

Reply:
I mean, passing `c("col1", "col2")` works but seems unintentional. I certainly didn't try to support it. I'm not sure if tidyselect is supposed to support character vectors either, if it's a bug, technically unsupported (so that it could go away at any time), supported. But I thought it could be good to document, because it is convenient.

Reply:
`c("col1", "col2")` working in tidyselect is intentional.  `vec <- c("col1", "col2"); ..... select(vec)` perhaps was intentional at some point, but now triggers an warning (but only like once per 8 hours or per session or something), because it's ambiguous: is this supposed to try to select column `vec` or the columns `col1, col2`?

We could potentially just remove mention of this possibility + the `:` discussion and instead just link again to tidyselect in "other tidy-select expression".

Reply:
Oh wait, I might be using bad tidyselect syntax with `!!vars` as well; the warning seems to recommend `all_of(vars)` or `any_of(vars)`.  Plus `!!` is typically for scalars... `!!!` might make more sense, but it's also not in the recommendations from the warning.